src/js/grapl-cdk/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# NOTE: this build is half-baked, and primarily supplies us with typechecks at CI time.
+# I haven't yet figured out a way to run the tests, primarily owing to the fact we need the ZIP'd artifacts
```

Comment:
I outlined two different potential approaches to this in the #infra channel. Otherwise, this is already better than what we've currently got so 💯

Reply:
^ for future reference:
https://grapl-internal.slack.com/archives/C017DKYF55H/p1599011803166100